test/iron-overlay-behavior.html

```diff
@@ -98,6 +125,31 @@
         overlay.close();
       }
 
+      function listenOnce(node, eventName, callback) {
+        var listener = function(event) {
+          node.removeEventListener(eventName, listener);
+          callback(event);
+        };
+        node.addEventListener(eventName, listener);
+      }
+
+      function pressTabAndChangeFocus(useShift, nodeToFocus) {
+        var TAB = 9;
+        if (useShift) {
+          // Limitation of MockInteractions for switching focus
+          var event = new CustomEvent('keydown', {
```

Comment:
Addressed in https://github.com/PolymerElements/iron-test-helpers/pull/45
Will refactor this once `modifiers` will be available